google-cloud-clients/google-cloud-datastore/src/main/java/com/google/cloud/datastore/spi/v1/HttpDatastoreRpc.java

```diff
@@ -66,7 +66,7 @@ public HttpDatastoreRpc(DatastoreOptions options) {
               + com.google.datastore.v1.client.DatastoreFactory.VERSION
               + "/projects/"
               + options.getProjectId();
-      clientBuilder = clientBuilder.projectId(null).projectEndpoint(fullUrl);
+      clientBuilder = clientBuilder.projectId(null).host(fullUrl);
```

Comment:
Do Not group all deprecation together, group them by their functionality, like this.

@ajaaym Please let me know If anything otherwise.